src/components/layout/dashboard/DashboardContentContainer.tsx

```diff
@@ -116,8 +116,8 @@ const DashboardContentContainer: React.FC<Props> = ({
           )}
           <Box
             sx={{
-              py: { xs: 1, sm: 2 },
-              px: { xs: 2, sm: 3, lg: 4 },
+              py: { xs: 2 },
+              px: { xs: 0, sm: 1, lg: 4 },
```

Comment:
Adding `xs: 0` here looks lovely on all of the screens that have a <Container> root component, but that turns out to not be everybody (see Assessment forms). I'll keep looking around to see where this needs to be added.